README.md

```diff
@@ -331,9 +329,16 @@ export default function ColoradoStateParks() {
 Take a look at the first line of code in this file: `import React from 'react'`.
 Here, we are referencing the React library's default export. The React library
 is located inside of the `node_modules` directory, a specific folder in many
-Node projects that holds packages of third-party code. Any time we are using
-code from an npm package, we must also import it in whatever file we're using it
-in.
+Node projects that holds packages of third-party code. In most cases, any time 
+we are using code from an npm package, we must also import it in whatever file 
+we're using it in. 
+
+As we learned in a previous lesson, for React specifically, it's no longer
+necessary to import it from the package into our component files. You'll notice 
+it has been omitted from the `MesaVerde.tsx` file, for example. 
+
+However, in most other cases, you will likely have to import packages when using
+their code. 
 
 > **Note**: The ability to import packages from the `node_modules` directory is
 > made possible in projects created with `create-react-app` by a tool called
```

Comment:
On line 353, recommending changing to: "keep our code modular, and **allow us to** use variables..."

On line 356, should be "... that **are** installed"